web/config/settings/settings_base.py

```diff
@@ -258,7 +258,7 @@
 
 MAX_EXPORT_ATTEMPTS = 3
 FORCE_DOCX_FOOTNOTES = True
-FORCE_DOCX_SECTIONS = False
+FORCE_DOCX_SECTIONS = True
```

Comment:
This will make footnotes and sections the default on stage/prod when it goes live.